### PR TITLE
Feature: Change the name of the game

### PR DESCRIPTION
**Pull request by Svenskapojkarna**
This pull reguest addresses the issue #2 

This feature adds a new bar on top of the action bar, where user can change the name of their game. Earlier the name was set as pelidata. Now the default name is Kiigame and it can be changed to any name. The name is added to the exported package. The name of the game is also feched from the name of the import package. If the user tries to export a game with an empty name, he/she will be notified that the name of the game cannot be empty.

Cypress tests are added to test the functionality of the name change.

In this pull request there is also a fix for broken CI build.
@evktalo could you add a review?

BR,
Aleksi Hytönen

### CODE REVIEW

File: kged/src/actions/global.js [hunk 1]
```diff
@@ -14,43 +14,49 @@ export const exportProject = (event) => {
...
-        zip.file('texts.json', textsToJSON);
-
-        zip.generateAsync({type: 'blob'}).then(content => {
-            saveAs(content, 'pelidata.zip');
-        });
+        if (state.global.gameName === ""){
+            alert('Pelin nimi ei voi olla tyhjä!')
+        }else if(window.Cypress){
+            alert(state.global.gameName)
+        }else{
```
Comment (evktalo) [inline kged/src/actions/global.js]: Code style is a little bit inconsistent with existing code.
```suggestion
        if (state.global.gameName === "") {
            alert('Pelin nimi ei voi olla tyhjä!')
        } else if (window.Cypress) {
            alert(state.global.gameName)
        } else {
```

File: kged/src/actions/global.js [hunk 2]
```diff
@@ -14,43 +14,49 @@ export const exportProject = (event) => {
...
-        zip.file('interactions.json', interactionsToJSON);
-        zip.file('texts.json', textsToJSON);
-
-        zip.generateAsync({type: 'blob'}).then(content => {
-            saveAs(content, 'pelidata.zip');
-        });
+        if (state.global.gameName === ""){
+            alert('Pelin nimi ei voi olla tyhjä!')
+        }else if(window.Cypress){
+            alert(state.global.gameName)
```
Comment (evktalo) [inline kged/src/actions/global.js]: Test should be written without requiring related logic in the code itself.

File: kged/cypress/integration/roomtest.js [hunk 3]
```diff
@@ -146,6 +146,28 @@ describe('inspector tests', function(){
...
+          })
+    })
+    it('changes the name of the game', () => {
+        const stub = cy.stub()
+        cy.get('.game-name').type('pelinimi')
+        cy.on('window:alert', stub)
+        cy.get('div').contains('Tallenna').click().then(() => {
+            expect(stub.getCall(0)).to.be.calledWith('pelinimi')      
+          })
+    })
```
Comment (evktalo) [inline kged/cypress/integration/roomtest.js]: Can we check the TitleBar element content instead?

File: kged/src/components/titlebar.jsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        this.onChangeHandler = this.onChangeHandler.bind(this)
+    }
+
+    onChangeHandler = event => {
+        event.preventDefault()
+        this.props.setName(event.target.value)
+    }
+
+    render(){
+        return(
```
Comment (evktalo) [inline kged/src/components/titlebar.jsx]: Small inconsistency with existing code style.

```suggestion
    render() {
        return (
```

**Comment by Svenskapojkarna** [comment]
Thanks for the review. I have changed the tests to check that the value of the input field is changed. Also, the inconsistencies have been fixed.

**Comment by evktalo** [comment]
@Svenskapojkarna I tested via the netlify preview link, and there's an issue; if you type in a name for the game, and press enter, it will reload the entire page. Shouldn't do this. This can be covered in the cypress test as well.

Also, I'd like to request that the game name label and the field would be on the same row, instead of label on top and the field below. Right now they take a lot of vertical space.

**Pull request merged**